### ISSUE TITLE
Geometry of the phantom for plan2

### ISSUE THREAD

**Issue by grzanka** [APTG/2022_DCPT_LET#51]
I was trying to cross-check total thickness of solid water phantom and a slab holding detectors.
When looking at the main [README.md](https://github.com/APTG/2022_DCPT_LET/blob/main/README.md) I find:

> 30 x 30 cm² slabs of Gammex solid water, 20 cm thick
> one 30 x 30 x 0.5 cm³ slab of PMMA which hold detectors of interest. This slab will always be positioned so its center plane (0.25 cm depth in the PMMA slab) will mark the measurement position.

This means that the total thickness should be 20 + 0.5 = 20.5 cm.

On the other hand in the [SHIELD-HIT12A geometry definition](https://github.com/APTG/2022_DCPT_LET/blob/main/data/sh12a/input/plan_2/geo.dat) for plan 2 I find:

```
  RPP    3     -15.0      15.0     -15.0      15.0     -2.25      17.5
```

That volume has thickness of 17.5 + 2.25 = 19.75 cm

The same applies to the [scoring file](https://github.com/APTG/2022_DCPT_LET/blob/main/data/sh12a/input/plan_2/detect.dat)

```
Geometry Mesh
        Name XZ_map             # Longitudinal map
        X -15.  15.   300
        Y  -1.   1.   1
        Z  -2.25 17.5 200

Geometry Mesh
        Name XY_map             # Lateral map
        X -15.  15.   300
        Y -15.  15.   300
        Z -0.25 0.25  1

Geometry Mesh
        Name Z_narrow           # Narrow scoring along Z (lat. equib)
        X  -1.  1.     1
        Y  -1.  1.     1
        Z  -2.25 17.5  200

Geometry Mesh
        Name Z_wide             # wide scoring along Z (no lat. equib)
        X  -15.  15.     1
        Y  -15.  15.     1
        Z  -2.25 17.5    200
```

I would expect that the Z extent should be from -2.25 to 18.25, broken down into:

- 2 cm of solid water slabs, from -2.25 to -0.25 cm
- 0.5 cm of PMMA slab holding detectors, from -0.25 cm to 0.25 cm
- 18 cm of solid water slabs, from 0.25 cm to 18.25 cm

If that is the case, then `geo.dat` and `detect.dat` files should be corrected by changing `17.5` to `18.25`. Also number of the bins should be adjusted from 200 to 205, to have bins of 1 mm thickness.

**Comment by grzanka** [comment]
This observation means that SH12A input files are inconsistent with scoring described here: https://github.com/APTG/2022_DCPT_LET/pull/50

**Comment by Villadslj** [comment]
I agree with the observation. The same discrepancy is not present in the topas file for plan2. So if this is correct, then I need to change those

**Comment by grzanka** [comment]
> I agree with the observation. The same discrepancy is not present in the topas file for plan2. So if this is correct, then I need to change those

Do you mean that TOPAS input files are inconsitent with description of the slab thickness ?
